dataflux_pytorch/benchmark/checkpointing/multinode/README.md

```diff
@@ -170,3 +170,30 @@ To run the script use the following command.
 ```shell
 python3 -m dataflux_pytorch.benchmark.checkpointing.multinode.train_async_save
 ```
+
```

Comment:
Wouldnt it be more appropriate to move the numbers to dataflux_pytorch/benchmark/checkpointing/multinode/README.md instead of after async_save ? 

I think each execution details followed by it's respective results would look better.